lib/server/ServerRenderingWorkflow.js

```diff
@@ -97,6 +97,22 @@ var ServerRenderingWorkflow = {
 
 };
 
+function promisify(method) {
```

Comment:
unfortunately i repeated because the one in this file has to use promise/domains. i figured it was easier to repeat the code than to pass in the promise object to all uses of the function like:

```js
promisify(func, require("promise/domains"))
```